### PR TITLE
Connect-database

### PR DESCRIPTION
**Pull request by hieunguyen64**

### CODE REVIEW

File: controller/ControllerCart.php [hunk 1]
```diff
@@ -1,13 +1,11 @@
 <?php
```
Comment (thienhophu) [inline controller/ControllerCart.php]: Rename to Cart.php

File: controller/ControllerCart.php [hunk 2]
```diff
@@ -20,13 +18,8 @@ public function __construct()
...
     {
-        //    echo "ham index..., in ra 10 sach ngau nhien";
-        $model = new ModelSach();
-        $datasach = $model->get10Sach();
-        $dataallsach = $model->getAllSach();
-        include 'View/home_index.php'; //load view
-    }
 
+    }
     public function f()
```
Comment (thienhophu) [inline controller/ControllerCart.php]: add extra line above this

File: controller/ControllerHome.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    public function index()
+    {
+        $model = new ModelBooks();
+        $dataallbooks = $model->getAllBooks();
+        include 'view/home_index.php'; //load view
+
+    }
+    public function f()
```
Comment (thienhophu) [inline controller/ControllerHome.php:25]: add extra line above this

File: controller/ControllerHome.php [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+            $this->f();
+        }
+    }
+
+    public function index()
+    {
+        $model = new ModelBooks();
+        $dataallbooks = $model->getAllBooks();
+        include 'view/home_index.php'; //load view
+
```
Comment (thienhophu) [inline controller/ControllerHome.php]: remove this line

File: controller/ControllerHome.php [hunk 5]
```diff
@@ -0,0 +1,28 @@
+<?php
+class ControllerHome
```
Comment (thienhophu) [inline controller/ControllerHome.php:3]: add extra line above this

**Review by thienhophu — CHANGES_REQUESTED** [review]

File: controller/ControllerCart.php (hunk 1 above)
Comment (hieunguyen64) [reply controller/ControllerCart.php]: done

File: controller/ControllerCart.php (hunk 2 above)
Comment (hieunguyen64) [reply controller/ControllerCart.php]: done

File: controller/ControllerHome.php (hunk 3 above)
Comment (hieunguyen64) [reply controller/ControllerHome.php:25]: done

File: controller/ControllerHome.php (hunk 4 above)
Comment (hieunguyen64) [reply controller/ControllerHome.php]: done

File: controller/ControllerHome.php (hunk 5 above)
Comment (hieunguyen64) [reply controller/ControllerHome.php:3]: done

**Pull request merged**